### DISCUSSION TITLE
Default startup pane/view

### DISCUSSION THREAD

**Discussion by razvan** [jesseduffield/lazygit#3189] [General]
Hey,

is it possible to change the default view (panel) on start up ? I would like to have LazyGit start with a maximized commits (reflog) pane and possibly the patch pane to the right.

Sure I can press `tab tab + +` and arrive there easily but it's sightly annoying for my use case. I work with many repositories at the same time, but most of the time I'm just interested in reading/searching the commit history.

I searched the config, discussions and issues and didn't find an answer.

Thanks.

**Comment by mark2185** [comment]
There is the option to open a certain panel by default, in your case that would be `lazygit log` (look at `lazygit --help` regarding `git-arg`), but that starts in the fully zoomed mode, so you'd need to press `_` after launching.

**Comment by vext01** [comment]
Similar question, is there a way to start in the git history for the current branch, thus emulating `tig`s default view?

You can do `lazygit branch` and hit <enter>, but is there a way to eliminate the extra <enter> keypress?

Thanks

**Comment by alcroito** [comment]
I created a feature request at https://github.com/jesseduffield/lazygit/issues/5877
